tests/test_examples.py

```diff
@@ -33,7 +33,11 @@ def test_mnist():
 
 def test_pylearn2():
     filename = 'unittest_markov_chain'
-    pylearn2_test('train', filename, 0, 3, False)
+    try:
+        pylearn2_test('train', filename, 0, 3, False)
+    except OSError:
```

Comment:
Haha, we can do that. I can try fixing it later, but it's not really a priority for me. It's probably got to do with `create_unbounded_method` or `property_` because the error says that `cPickle` fails trying to pickle a function object.